bugbug/model.py

```diff
@@ -93,6 +93,11 @@ def classification_report_imbalanced_values(
 
 def print_labeled_confusion_matrix(confusion_matrix, labels, is_multilabel=False):
     confusion_matrix_table = confusion_matrix.tolist()
+    try:
+        confusion_matrix_table.pop(labels.index("NOT_CLASSIFIED"))
+    except ValueError:
+        pass
```

Comment:
When is the exception thrown?

Reply:
Is it for when there is no confidence threshold?

Reply:
If so, it'd be better to do:
```
if "___NOT_CLASSIFIED___" in labels:
    confusion_matrix_table.pop(labels.index("___NOT_CLASSIFIED___"))
```

And a comment before the if to explain that we don't want to show the "Not classified" row in the table output.